custom_model_runner/datarobot_drum/drum/gpu_predictors/triton_predictor.py

```diff
@@ -114,12 +114,14 @@ def readiness_probe(self):
         return self.health_check()
 
     def health_check(self):
+        # TODO: would be good to respond with 513 status code if Triton server has crashed or
+        #   if we can detect some other terminal error relating to loading the model provided.
```

Comment:
@alexromanyuk from your experience, is there an easy way to add this to Triton env? My first question would be, how does Triton behave if it sees an invalid model in its model directory:
- Does it terminate
- if not, are there any health routes that can differentiate between it starting up vs. being ready but having no models to serve?
- any other options to detect a terminal error?

Reply:
seems I missed this question. may be not relevant now, but as far as I remember - Triton will terminate if there are no models to serve or that single model can't be loaded.

> are there any health routes that can differentiate between it starting up vs. being ready but having no models to serve?

The `/health/live` - server started and the `/health/ready` - models loaded and ready for inference. see here 
https://docs.nvidia.com/deeplearning/triton-inference-server/archives/triton_inference_server_1120/triton-inference-server-guide/docs/http_grpc_api.html#health

Reply:
and this one
> Performing an HTTP GET to /api/status returns status information about the server and all the models being served. Performing an HTTP GET to /api/status/\<model name\> returns information about the server and the single model specified by <model name>.